src/main/java/Model/Database/Interaction/I_Measurements.java

```diff
@@ -86,6 +86,53 @@ public static T_Measurement retrieve(Connection conn, PreparedStatement ps, Resu
         return tm;
     }
 
+    public static ArrayList<T_Measurement> retrieveFilteredAll(Connection conn, PreparedStatement ps, ResultSet rs, int sensorId) throws SQLException {
+
+        // No Filter is being used
```

Comment:
zase ta ista otazka, preco je tu moznost ziadne filtrovania, ked to je potom vlastne metoda retrieveAll? ako si predstavujes ze sa pouzije tato metoda bez filtrov. to nie je kritika, len tomu nerozumiem